flink-runtime/src/main/java/org/apache/flink/runtime/scheduler/exceptionhistory/ExceptionHistoryEntry.java

```diff
@@ -63,21 +66,27 @@ public static ExceptionHistoryEntry create(AccessExecution failedExecution, Stri
         return new ExceptionHistoryEntry(
                 failure.getException(),
                 failure.getTimestamp(),
+                failure.getLabelsFuture(),
                 taskName,
                 failedExecution.getAssignedResourceLocation());
     }
 
     /** Creates an {@code ExceptionHistoryEntry} that is not based on an {@code Execution}. */
     public static ExceptionHistoryEntry createGlobal(Throwable cause) {
```

Comment:
Can we mark this method as deprecated and maybe label it with an issue that will get rid of it?

Reply:
The same applies to other factory methods that ignore labels

Reply:
This is actually tracked by the follow-up FLINK-31892 so I believe we are good here (all other methods include labels)
https://github.com/apache/flink/pull/22587/commits/99f41cfb94b20162cce28f7e9ec5662afe385689#diff-96bec50fecaaa9b559479b5f90e508fb6185a9a1886f295f0dbe3d0567794e38R74-R75